layouts/index.html

```diff
@@ -8,18 +10,30 @@
       <!-- Register BEGIN -->
       <section class="usa-grid page-inner" id="main-content">
         <header>
-          <h1 class="usa-display">{{ ( index $.Site.Data.translations $.Site.Params.lang ).homepageheader }}</h1>
+          <h1 class="usa-display">{{ $translation.homepage.header }}</h1>
```

Comment:
Because typing out `( index $.Site.Data.translations $.Site.Params.lang[uage] )` was starting to get _very verbose_.